src/iotjs_env.h

```diff
@@ -18,13 +18,15 @@
 
 #include "uv.h"
 
+typedef struct {
+  bool debugger_wait_source;
+  int debugger_port;
```

Comment:
It looks better to remove `debugger_` prefix here.

Reply:
I agree. No need to prefix variables inside a struct.

Reply:
Yeah, that's actually right, I didn't change them after moving them to the structure.